tmail-backend/jmap/extensions-api/pom.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.linagora.tmail</groupId>
+        <artifactId>tmail-backend</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>jmap-extensions-api</artifactId>
+    <name>Twake Mail :: JMAP :: Extensions :: API</name>
+
+    <properties>
+        <uuid-creator.version>5.3.7</uuid-creator.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>james-server-data-jmap</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>james-server-guice-configuration</artifactId>
```

Comment:
Is it needed?

Reply:
According to `mvn dependecy:analyze` yes we need it. It is detecting testing-base as unused probably because it is a transitive dependency of another dependency but I wanna keep it explicit.

![image](https://github.com/user-attachments/assets/04da7ff8-5a1c-4c03-9cc8-1a1097a6861d)

Reply:
Removed it and the code compiled fine; `dependecy:analyze` is not reliable unfortunately.